minicluster/pom.xml

```diff
@@ -19,8 +19,30 @@
     <findbugs.path>${project.parent.basedir}/build/findbugs/</findbugs.path>
   </properties>
 
+  <build>
```

Comment:
NIT: to keep consistency with other pom files, could you move this `<build>` session after `<dependencies>` session?